src/Features/CSharp/Portable/InlineHints/CSharpInlineParameterNameHintsService.cs

```diff
@@ -30,7 +32,7 @@ public CSharpInlineParameterNameHintsService()
         protected override void AddAllParameterNameHintLocations(
              SemanticModel semanticModel,
              SyntaxNode node,
-             ArrayBuilder<(int position, IParameterSymbol? parameter, HintKind kind)> buffer,
+             ArrayBuilder<(int position, SyntaxNode argument, IParameterSymbol? parameter, HintKind kind)> buffer,
```

Comment:
I'd make AbstractInlineParameterNameHintsService generic on a TArgumentSyntax type so that we can be sure we have the right syntax node in tehse cases.

Reply:
ignore this.  just don't pass a node around, pass the text of the argument if it is an identifier.